src/frontend/js/modules/header/header.js

```diff
@@ -114,6 +114,7 @@
           };
 
           scope.domainId = session.domain._id;
+          scope.apiURL = window.openpaas && window.openpaas.OPENPAAS_API_URL || '';
```

Comment:
rename it `baseUrl` like here https://github.com/linagora/esn-frontend-common-libs/blob/43972a3af0775899526b030cb32e94b011f23300/src/frontend/js/modules/http.js#L102